javascript/reactjs-todo-login-widget/.env.example

```
@@ -1,17 +1,14 @@
-DEBUGGER_OFF=true
-DEVELOPMENT=
-JOURNEY_LOGIN=
-JOURNEY_REGISTER=
-PORT=
-REALM_PATH=
-REST_OAUTH_CLIENT=
-REST_OAUTH_SECRET=
 WEB_OAUTH_CLIENT=
-CENTRALIZED_LOGIN=
-SERVER_TYPE=
+SCOPE='openid profile email'
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SCOPE key should go before the SERVER_URL key

(UnorderedKey)